### ISSUE TITLE
[NODES] Crop resize bug

### ISSUE THREAD

**Issue by GUY-BBS** [olive-editor/olive#1817]
**38372791** 

**Platform** 

**I am trying to crop a 618 x 233-pixel caption on a 1960 x 1080 timeline. The crop parameters do not appear in the Parameter Editor window unless I select the clip on the timeline after adding the node.  I don't recollect this happening on earlier commits? When I adjust the crop parameters either in the Parameter Editor window or by grabbing the handles in the Sequence Viewer window, the caption resizes to fill the frame.  The video shows the effect. **

**Additional Information / Output**

https://user-images.githubusercontent.com/49677111/148091894-0d3c6a49-fa04-4aea-93f9-b24eb4b27258.mp4

**Comment by itsmattkc** [comment]
These are two separate issues. To clarify:

- [ ] Newly added node doesn't appear in parameter view until clip is re-selected.
- [ ] Crop + Transform exhibit strange behavior 

These are both bugs and should be resolved, however I will mention that you should be able to "work around" the second one by placing the crop after the transform.

**Comment by GUY-BBS** [comment]
I have discovered a similar bug as the problem with the crop.
I have been attempting to create a colour filter effect using the node arrangement below.
 
![Colour Filter 1](https://user-images.githubusercontent.com/49677111/150182094-5cf8b58a-4cea-45d6-9154-506a63a6a210.jpg)
 I could not get this to work with the transform scale set at 100%
![Colour Filter 3](https://user-images.githubusercontent.com/49677111/150182444-ad3019e9-65b1-48c4-9ca6-dc777b785d52.jpg)
Only when I change the value to anything other than 100 did the colour filter work as planned.
![Colour Filter 2](https://user-images.githubusercontent.com/49677111/150182729-faca3555-422b-4ae7-8345-6fb2b0a04bd5.jpg)
As it might well be related to the crop problem, I have posted it here.
